pkg/logger/logger_test.go

```diff
@@ -120,3 +121,63 @@ func TestDebugPayload(t *testing.T) {
 		})
 	}
 }
+
+func TestGetLogLevelByNameWithSuffix(t *testing.T) {
+	tmpdir, err := os.MkdirTemp(os.TempDir(), "receptor-test-suffix*")
```

Comment:
```suggestion
	tmpdir, err := os.MkdirTemp("", "receptor-test-suffix*")
```

When using `os.mkdirTemp` we're telling Go we want a temporary file. We shouldn't need to tell it where. But this may be obsolete (see comment on `io.Writer` and `bytes.Buffer` below so don't rush to implement it.